### ISSUE TITLE
Test ppc64el on Travis

### ISSUE THREAD

**Issue by nikic** [php/php-tasks#13]
Travis also supports ppc64el architecture: https://docs.travis-ci.com/user/multi-cpu-architectures/

We should test it, and drop one of our amd64 jobs instead.

**Comment by nikic** [comment]
Work in progress: https://github.com/php/php-src/pull/5791

**Comment by nikic** [comment]
I guess this is a moot point now that we no longer use Travis.

It would be nice to find a new way to test on non-x86 architectures.

**Comment by kocsismate** [comment]
Apparently, there is a GitHub action for this: https://github.com/marketplace/actions/run-on-architecture

**Comment by nikic** [comment]
Yeah, I found that one as well. It's based on qemu, so maybe slow... Probably worth giving it a try though. At least arm64 seems like something we should be testing.
